install/kubernetes/cilium/templates/hubble/tls-cronjob/_job-spec.tpl

```diff
@@ -21,6 +21,10 @@ spec:
               drop:
               - ALL
             allowPrivilegeEscalation: false
+          {{- with .Values.certgen.resources }}
+          resources:
+          {{- toYaml . | nindent 10 }}
+          {{- end }}
```

Comment:
Drive-by comment, please apply the same changes to the clustermesh certgen instance as well, to keep them aligned: https://github.com/cilium/cilium/blob/main/install/kubernetes/cilium/templates/clustermesh-apiserver/tls-cronjob/_job-spec.tpl

Reply:
Added